### ISSUE TITLE
Placing bets fails 100% of the time

### ISSUE THREAD

**Issue by villiers-s** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#262]
**Describe the bug**
Ever since updating from 40dcf78 to 76679e9, placing bets fails consistently. Each bet fails with:

```
INFO - [make_predictions]: 😥  Oh no! The event is not active anymore! Current status: LOCKED
```
or
```
ERROR - [make_predictions]: 🍀  Failed to place bet, error: EVENT_NOT_ACTIVE
```

It looks like it tries placing the bets too late, but it's hard to tell exactly what the problem is:

```
02/09/21 20:36:05 - INFO - [on_message]: ⏰  Place the bet after: 299.99s for: EventPrediction(...
02/09/21 20:41:05 - INFO - [make_predictions]: 🍀  Going to complete bet for EventPrediction(...
02/09/21 20:41:05 - INFO - [make_predictions]: 🍀  Place 6.01k channel points on: Yes (PINK), Points: 1M, Users: 59 (66.29%), Odds: 1.14 (87.72%)
02/09/21 20:41:06 - ERROR - [make_predictions]: 🍀  Failed to place bet, error: EVENT_NOT_ACTIVE
```

**To Reproduce**
1. Run the channel points miner for a channel that has bets
2. Wait for a bet
3. Placing the bet will fail because the bet is no longer active

**Expected behavior**
The bet should be successfully placed.

**Desktop (please complete the following information):**
 - OS: mac
 - Python version 3.9.5

**Additional context**
I'm not using any custom values for delay in the bet settings and I didn't change any settings after updating.

**Comment by RakSrinaNa** [comment]
As you pointed out `Placing the bet will fail because the bet is no longer active`. This is most likely due because a moderator closed the predictions before the original timer.
The bot by default places its predictions a few seconds before the end of the original timer. If you wish to change that look at https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2#delaymode and for example set it to maybe always place bets 1 mins after the predictions are open.

**Comment by villiers-s** [comment]
I find it very unlikely that _all_ the bets are closed early. 103 out of 103 bets failed since I updated.

**Comment by RakSrinaNa** [comment]
Well if you got the reasons `LOCKED` and `EVENT_NOT_ACTIVE` it means it was closed before the bed is placed.
Either you'll have to monitor what is happening directly on twitch, or try with a shorter delay In the example you gave it waits for 299s (so nearly 5mins), try reducing it to like 1 mins since bet is open and see if the errors are gone (in which case it means bets are closed earlier).

**Comment by villiers-s** [comment]
My current theory, that I'll try to test later today, is that there's something wrong with the default delay settings, because I haven't set any custom delay since the delay settings were introduced.

As far as I understand it, the default is to place the bet 6 seconds before the end of the bet. With that one example, it's very likely that the bet was set for 5 minutes, so it should have waited about 294 seconds before placing the bet and not 299 seconds.

I'll report back as soon as I have done some tests.

**Comment by RakSrinaNa** [comment]
This is indeed the behavior and the 6 (of the 6 seconds) comes from the default value of the config key `streamer_settings.bet.delay`.

**Comment by villiers-s** [comment]
Found the issue, turns out the default settings were never applied. I opened a PR to fix this #264
